examples/step-31/step-31.cc

```diff
@@ -952,9 +962,9 @@ namespace Step31
     // Trilinos matrices store the sparsity pattern internally, there is no
     // need to keep the sparsity pattern around after the initialization of
     // the matrix.
-    stokes_block_sizes.resize (2);
-    stokes_block_sizes[0] = n_u;
-    stokes_block_sizes[1] = n_p;
+    stokes_partitioning.resize (2);
+    stokes_partitioning[0] = complete_index_set (n_u);
+    stokes_partitioning[1] = complete_index_set (n_p);
```

Comment:
Oh, I see. I had thought that the old constructor takes a vector of numbers for the block sizes, that the new constructor call should take index sets that cleanly partition the entire space into blocks. But all they do is that they partition the blocks onto processors. My bad -- ignore the comment.